bower.json

```diff
@@ -22,6 +22,11 @@
     "test",
     "tests"
   ],
+  "dependencies": {
+    "angular": "~1.4.4",
+    "angular-sanitize": "~1.4.4",
```

Comment:
I was using your formatter, and it was going great, but as I updated to a bower build process, the tools that pull scripts from bower components (https://github.com/ck86/main-bower-files) rely on the `dependencies` property in bower.json to figure out what order to include the JS files. 

Because your library references "angular", angular must be included **before** the JS for this plugin. Listing angular as an explicit dependency makes sure that it is included first.  Otherwise, I get an error "angular is not defined".